Resources/Prototypes/Entities/Items/Consumables/drinks_cups.yml

```diff
@@ -237,3 +237,23 @@
     visuals:
     - type: DrinkFoodVisualizer2D
       steps: 4
+
+# Transformable container - normal glass
+- type: entity
+  name: Drinking glass
+  parent: DrinkBaseCup
+  id: DrinkGlass
+  components:
+  - type: Sprite
+    sprite: Objects/Drinks/glass_clear.rsi
+  - type: Icon
+    sprite: Objects/Drinks/glass_clear.rsi
+  - type: Solution
+    fillingState: glass
+    maxVol: 50
+    caps: 16
+  - type: Drink
+
+  - type: Pourable
```

Comment:
Components should be un-separated, right?

Reply:
u right u right